consensus/istanbul/core/preprepare.go

```diff
@@ -55,9 +57,20 @@ func (c *core) handlePreprepare(msg *message, src istanbul.Validator) error {
 		return errFailedDecodePreprepare
 	}
 
+	// If round > 0, handle the ROUND CHANGE certificate.
```

Comment:
Should we require that the ROUND CHANGE certificate is empty if round == 0?

Reply:
Probably should to keep it cleaner, but I don't think it hurts either way.